helper/include/odp/helper/cli.h

```diff
@@ -42,6 +45,12 @@ typedef struct {
 	const char *address;
 	/** TCP port. Default is 55555. */
 	uint16_t port;
+	/**
+	 * User defined command function. If set, this function is called when
+	 * the CLI client invokes the command "user". If not set (set to NULL),
+	 * then the CLI command "user" does not exist. Default is NULL.
+	 */
+	odph_cli_user_cmd_func_t user_cmd_fn;
```

Comment:
I'm thinking it would make sense to allow a user to add multiple custom commands.